plugins/out_syslog/syslog_conf.h

```diff
@@ -31,7 +31,6 @@ struct flb_syslog {
     flb_sockfd_t fd;
     struct flb_upstream *u;
     flb_sds_t mode;
-    int format;
```

Comment:
Please don't remove this variable and can you modify data type ? `int` -> `flb_sds_t` .